e2etests/orchestration/common.go

```diff
@@ -131,6 +131,33 @@ func (h *DockerHelper) RunContainer(img string, hostPort int) (string, error) {
 	return createRes.ID, nil
 }
 
+func (h *DockerHelper) PullLogs(id string) error {
```

Comment:
what logs are being pulled here?

Reply:
these are the containers logs which contain the host orchestrator logs.